src/components/select/filterable-select/index.d.ts

```diff
@@ -48,6 +49,10 @@ export interface FilterableSelectProps {
   onBlur?: () => void;
   /** Callback function for when the key is pressed when focused on Select Textbox. */
   onKeyDown?: () => void;
+  /** True for default text button or a Button Component to be rendered */
+  listActionButton?: boolean | typeof Button;
```

Comment:
This doesn't have the option to be boolean in the propTypes in the component file